components/core/src/package/install.rs

```diff
@@ -171,6 +171,7 @@ impl PackageInstall {
         let mut run_path = String::new();
         for path in try!(self.paths()) {
             run_path.push_str(&path.to_string_lossy());
+            run_path.push(':');
```

Comment:
This will cause a trailing `:` at the end of the path. If we check to see if we're on the last element and skip this bit that might be a wise. You could do that with something like:

```
for (i, path) in try!(self.paths()).enumerate() {
    run_path.push_str(&path.to_string_lossy());
    if i < self.paths().len() {
        run_path.push(":");
    }
    idents.insert(self.ident().clone());
}
```